### PR TITLE
upgrade to smartsense 1.3.0 which is now the default in Ambari 2.4

### PR DESCRIPTION
**Pull request by seanorama**
- consolidate to a single init.sls since the changes are only needed on the server, not the worker nodes
- remove blueprint gateway configurations
- disable gateway service, but keep gateway config (hst now has an embedded gateway which uses the file)
- ensure hst service is enabled at next boot
- minor change to smartsense.account.name

### CODE REVIEW

**Comment by seanorama** [comment]
@biharitomi to review

File: orchestrator-salt/src/main/resources/salt/salt/smartsense/init.sls [hunk 1]
```diff
@@ -1,27 +1,6 @@
...
-/etc/smartsense/conf/security.py.diff:
-  file.managed:
-    - makedirs: True
-    - source: salt://smartsense/scripts/security.py.diff
-    - mode: 755
-
-/etc/smartsense/conf/smartsense-agent-patch.sh:
-  file.managed:
-    - makedirs: True
-    - source: salt://smartsense/scripts/smartsense-agent-patch.sh
```
Comment (akanto) [inline orchestrator-salt/src/main/resources/salt/salt/smartsense/init.sls]: Patch script and the patch file can be deleted as well.

File: orchestrator-salt/src/main/resources/salt/salt/smartsense/gateway-init.sls [hunk 2]
```diff
@@ -10,8 +10,6 @@ nstall-smartsense-hst:
...
       - pkg: smartsense-hst
 
-start-gateway:
-  cmd.run:
-    - name: hst gateway start
-    - watch:
-      - file: /etc/hst/conf/hst-gateway.ini
\ No newline at end of file
+hst-gateway:
+  service.disabled:
```
Comment (akanto) [inline orchestrator-salt/src/main/resources/salt/salt/smartsense/gateway-init.sls]: this will disable hst-gateway, isn't it managed by Ambari?

Comment (seanorama) [inline orchestrator-salt/src/main/resources/salt/salt/smartsense/gateway-init.sls]: @akanto unfortunately, it's still not managed by Ambari.

**Comment by seanorama** [comment]
Added another commit to consolidate the salt work and remove scripts.

**Comment by akanto** [comment]
LGTM

**Pull request merged**